### DISCUSSION TITLE
Sideloading  /new JIT possibilities

### DISCUSSION THREAD

**Discussion by rquinn2** [holzschu/a-shell#767] [General]
There is the obviously negative of this splitting the app into two verisons, but it seems to have worked fine for UTM. 

But are there any upsides? What could be added to a "non-app store" versin of a-shell

**Comment by holzschu** [comment]
bash and emacs are the first things that come to mind.
For the downsides: it would be only for iPhones, not iPads. And I (currently) don't have a device on which I could test that potential new app.

**Reply by TransBudPol** [reply]
status of sideloading on ipad changed, and i think, a-shell users likely know how to sideload .ipa. I-think, jit or troll/jb version would change experience of using iPad as daily drive
